arrows/burnout/CMakeLists.txt

```diff
@@ -35,8 +31,12 @@ kwiver_add_library( kwiver_algo_burnout
 
 target_link_libraries( kwiver_algo_burnout
   PUBLIC               vital_algo
-  PRIVATE              ${burnout_linked_libs}
-                       vil vidtk_descriptor
+  PRIVATE              vidtk_descriptor vidtk_utilities_no_process vidtk_pipelines
```

Comment:
I've mentioned this for years now, but it's not correct for kwiver to depend on vidtk since kwiver is a hard requirement for vidtk. It would be good to find a way to migrate the required vidtk functionality into kwiver arrows, or make this arrow into a kwiver plugin, but even the latter leaves you stuck on an old vidtk.

Obviously it's not holding up this branch but it's going to cause you issues before long.